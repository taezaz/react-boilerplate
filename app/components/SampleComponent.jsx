// @flow
import React from 'react';
import type { SampleComponentProps } from './types';

const SampleComponent = (props: SampleComponentProps) => (
    <div>{props.text}</div>
);

export default SampleComponent;
