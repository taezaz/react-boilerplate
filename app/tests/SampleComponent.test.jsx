// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import SampleComponent from '../components/SampleComponent';

describe('Sample component', () => {
    it('should match snapshot', () => {
        const component = renderer.create(<SampleComponent text="test" />);
        expect(component).toMatchSnapshot();
    });
});
