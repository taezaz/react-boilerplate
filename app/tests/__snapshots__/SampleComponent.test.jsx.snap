// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample component should match snapshot 1`] = `
<div>
  test
</div>
`;
